packages/turf-rhumb-destination/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@turf/destination",
+  "version": "4.3.0",
```

Comment:
For new modules, define them as `4.0.0` (base number of major release) because whenever I publish with Lerna it will automatically bump up the appropriate version (using the same version number as the next release will cause a version conflict with Lerna).